### PR TITLE
Update Wix.xml

### PR DESCRIPTION
**Pull request by koops76**
#11114

### CODE REVIEW

File: src/chrome/content/rules/Wix.xml [hunk 1]
```diff
@@ -1,11 +1,4 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://archive.wix.com/ => https://archive.wix.com/: (35, 'Unknown SSL protocol error in connection to archive.wix.com:443 ')
-Fetch error: http://dns.lc.wix.com/ => https://dns.lc.wix.com/: (28, 'Connection timed out after 20001 milliseconds')
-Fetch error: http://www.dns.lc.wix.com/ => https://www.dns.lc.wix.com/: (6, 'Could not resolve host: www.dns.lc.wix.com')
-Fetch error: http://publicorigin.wix.com/ => https://publicorigin.wix.com/: (6, 'Could not resolve host: publicorigin.wix.com')
```
Comment (J0WI) [inline src/chrome/content/rules/Wix.xml]: Please add comments for the removed hosts (if they still exist)

File: src/chrome/content/rules/Wix.xml [hunk 2]
```diff
@@ -36,79 +29,39 @@ Fetch error: http://publicorigin.wix.com/ => https://publicorigin.wix.com/: (6,
...
 
+	<rule from="^http://tracking\.wix\.com/" to="https://wix.go2cloud.org/" />
 
-	<rule from="^http://tracking\.wix\.com/"
-		to="https://wix.go2cloud.org/" />
+	<rule from="^http:"	to="https:" />
 
-	<rule from="^http:"
-		to="https:" />
+	<exclusion pattern="^http://www\.wix\.com/+(?!favicon\.ico|(?:my-account|signin|support)(?:$|[?/]))" />
```
Comment (J0WI) [inline src/chrome/content/rules/Wix.xml]: They do no longer redirect to http, please remove all obsolete exclusions/tests.

File: src/chrome/content/rules/Wix.xml (hunk 1 above)
Comment (koops76) [inline src/chrome/content/rules/Wix.xml]: @J0WI Done.

File: src/chrome/content/rules/Wix.xml (hunk 2 above)
Comment (koops76) [inline src/chrome/content/rules/Wix.xml]: @J0WI Done.

**Pull request merged**